packages/storybook/stories/search-contextual.mdx

```diff
@@ -0,0 +1,43 @@
+import { SearchContextual } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Stories, Story, Canvas } from '@storybook/blocks';
+import * as SearchContextualStories  from './search-contextual.stories';
+
+<Meta of={SearchContextualStories} />
+
+# Search Contextual (Live Search)
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+Live search allows users to filter elements below the component directly by typing into the field. The variant does not have a popup list that appears since users don’t have to make a selection to filter the options. 
```

Comment:
Je me demande de la pertinence de ce texte: "_The variant does not have a popup list that appears since users don’t have to make a selection to filter the options._". Ce n'est pas une variante et ca ne vient pas vraiment décrire le composant. Je vois ça plus comme un "_When not to use_".